### DISCUSSION TITLE
GEFS Precipitation prob

### DISCUSSION THREAD

**Discussion by phillipseamore** [open-meteo/open-meteo#238] [General]
It seems that you are downloading control + each individual member, couldn't you save a ton of requests by using the geavg files instead?

**Comment by patrick-zippenfenig** [comment]
It is possible to use standard deviation and average from GFS ensembles to "estimate" probability of rain events >1mm/hour. This distributional probability approximation does not work very well with "only" 30 members and also the shape of distribution is unknown (I could assume Gauss).

The GFS ensemble standard deviation is based on accumulated precipitation, which makes it harder to use directly. I am not even sure if it is possible to deaccumulate standard deviation from 6 hourly accumulations to 3h. I am to lazy to do the math now :D

Ensemble models are also available from many other national weather services. I want to add all ensemble modes with individual members as an additional API. This will however be a larger task.
